propeoplemd/cibox
# Move GitHub logic into cibox-project-builder role

Steps for review:
1) Clone repo
2) Run `./repository.sh`
3) Make sure it generates folder `FRESH_REPOSITORY` in root folder
4) Make sure this folder is excluded from git cache
5) Make sure there are no regressions

*Note:* `run.sh` has been renamed to `provision.sh`
